### DISCUSSION TITLE
Safari on Montery redirects .test to search engine [Version 15.1 (17612.2.9.1.20)]

### DISCUSSION THREAD

**Discussion by raphaelmsr** [laravel/valet#1147] [Q&A]
Hey all ! Valet user for years now, awesome package!
I believe the exposure is better opening a ticket here than on Apple's forums that will never be addressed.

Since I updated I can't access easily .test domains in my Safari. It keeps redirecting to my search engine (e.g. DuckDuckGo)
Does someone encounter the same problem ? Is there a fix ?

Thanks

**Comment by ihorvorotnov** [comment]
I'm on Version 15.1 (17612.2.9.1.20) and Monterey 12.0.1 as well, everything works fine. After upgrading macOS I upgraded all packages installed via Homebrew and did:
```
composer global update
valet install
```

**Comment by rodriguezcommaj** [comment]
I can confirm the same issue. Have tried digging into multiple settings in Safari and MacOS without success. Valet works perfectly on both Chrome and Firefox, but not Safari. It redirects to whatever search engine is selected as the default in Safari. 

I've even tried changing the domain using `valet domain whatever` without any success. Again, works in non-Safari browsers only.

**Comment by jools-r** [comment]
Not sure if this offers any help in tracking it down but what format do the URLs you're trying to access have? I have the same basic OS/Safari constellation and valet works successfully for me except for certain *potentially incorrect* URL combinations, e.g. the following work fine:

```
http://www.domain.test/
http://www.domain.test/path/parts
http://www.domain.test/?q=search-term
```

but I see what you describe when there's an url query but no trailing slash, e.g.

```
http://www.domain.test?q=search-term -> redirects to search engine
```

which happens sometimes when I mistype...

**Comment by raphaelmsr** [comment]
For me it's coming straight from the history so no mistakes are possible
Straight up starting to type down my domain, down arrow, enter key -> DuckDuckGo

**Comment by macCesar** [comment]
Same problem here with Safari 15.1 and Mac OS Monterey !!!

Before I could type `nameofproject.test` in Safari and worked as expected.

Now, I have to include `http://` in order to work!! 🤦🏻‍♂️

But it only works with base URLs.. If I need to access, `/admin` ( http://nameofproject.test/admin ) ... it goes straight to google search results. 😤

**Comment by rodriguezcommaj** [comment]
Found a workaround using [localtest.me](http://readme.localtest.me). Updating Valet's domain to `localtest.me` got things working for me without having to type `http://` every single time. 

Just run `valet domain localtest.me` and wait for it to update itself and it should work.

**Comment by adriaanzon** [comment]
Another workaround is to hold Control while pressing Enter.

**Comment by nickdekruijk** [comment]
> Another workaround is to hold Control while pressing Enter.

Doesn't seem to do anything different for me...

**Comment by pindab0ter** [comment]
Using a trailing slash (domain.test/) has consistently worked for me, not even requiring the protocol prefix.
